### PR TITLE
Python 5 cards

### PR DESCRIPTION
**Pull request by ohwi**
I've done with 5 cards.

I ran unittest with cpp version of evaluate_5cards as a ground truth and passed.

Tables are copied from C code.

One can understand the way of building tables by test_tables. However, the code need to be refactored.

### CODE REVIEW

File: python/test/test.py [hunk 1]
```diff
@@ -0,0 +1,43 @@
+from ctypes import *
+import unittest
+from evaluator5 import evaluate_5cards
```
Comment (HenryRLee) [inline python/test/test.py]: Import `evaluator5` doesn't seem to work unless I move all the source files and the test file to the same folder.

Would you be able to fix this part? I found this [repo](https://github.com/navdeep-G/samplemod/blob/master/tests/context.py) might be helpful.

File: python/test_tables/test_tables.py [hunk 2]
```diff
@@ -0,0 +1,395 @@
+import unittest
+from itertools import combinations, combinations_with_replacement
+
+from dptables import *
```
Comment (HenryRLee) [inline python/test_tables/test_tables.py]: These might also meet the `No module named ...` errors.

File: python/test/test.py [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+
+
+# TODO: Find a proper way to include
+ph_eval = cdll.LoadLibrary("../../cpp/build/libpheval.dylib")
```
Comment (HenryRLee) [inline python/test/test.py]: Yes, this is a problem. I'll see what I can do. Don't worry about it yet.

**Comment by HenryRLee** [comment]
It looks pretty good. Thanks for the effort.

I made some comments about the unavailable `syspath` in the test files. After fixing that, I think we can merge this PR and move to next ones.

**Comment by ohwi** [comment]
Sorry, I worked with PyCharm using mark directory as a source root tool and forgot to turn off.

I added __init__.py to all folders and inserted path of top folder.

**Comment by HenryRLee** [comment]
I am still struggling on how to run the test code. Would you please advice on how to do this? It'd be better if you could add a README file in the `python` root folder to tell people how to use it.

**Comment by ohwi** [comment]
Sorry for splitting test codes... I added bash script to run the test and README.

**Comment by HenryRLee** [comment]
I removed the `hashtable6` dependencies from the `hashtable.py` file, which is breaking.

Other than that, everything is great!

Thank you, and keep the good job on the rest of the evaluators.

**Pull request merged**